wand/frontend/wand_influx_db.py

```diff
@@ -5,14 +5,10 @@
 
 import influxdb
 
-from artiq.tools import init_logger
-from artiq.protocols.pc_rpc import Client as RPCClient
+from sipyco.pc_rpc import Client as RPCClient
 
-# verbosity_args() was renamed to add_common_args() in ARTIQ 5.0; support both.
-try:
-    from artiq.tools import add_common_args
-except ImportError:
-    from artiq.tools import verbosity_args as add_common_args
+# verbosity_args() was renamed to add_common_args() in ARTIQ 5.0
```

Comment:
can you remove this comment -- it doesn't make sense to track these changes in the comments?